### PR TITLE
Adding OpenShift security patch

### PR DESCRIPTION
**Pull request by TomRoSystems**

### CODE REVIEW

**Comment by TomRoSystems** [comment]
> why didn't we just pass the config object to this function?

I thought that it would be better to pass as minimal information as necessary...
Do you want me to change it?

File: .ci/create-release-github.sh [hunk 1]
```diff
@@ -7,4 +7,5 @@ OPERATOR_VERSION=$(git describe --tags)
 gh release create \
     -t "Release ${OPERATOR_VERSION}" \
     "${OPERATOR_VERSION}" \
-    'dist/splunk-otel-operator.yaml#Installation manifest for Kubernetes'
+    'dist/splunk-otel-operator.yaml#Installation manifest for Kubernetes' \
+    'dist/splunk-otel-operator-openshift.yaml#Installation manifest for Kubernetes for OpenShift'
```
Comment (owais) [inline .ci/create-release-github.sh:11]: `     'dist/splunk-otel-operator-openshift.yaml#Installation manifest for OpenShift'`

Comment (owais) [inline .ci/create-release-github.sh:11]:
```suggestion
    'dist/splunk-otel-operator-openshift.yaml#Installation manifest for OpenShift'
```

File: Makefile [hunk 2]
```diff
@@ -44,7 +45,7 @@ endif
 KUBE_VERSION ?= 1.21
 KIND_CONFIG ?= kind-$(KUBE_VERSION).yaml
 
-ensure-generate-is-noop: VERSION=$(VERSION)
+# ensure-generate-is-noop: VERSION=$(VERSION)
```
Comment (owais) [inline Makefile:48]: remove

File: versions.txt [hunk 3]
```diff
@@ -5,4 +5,4 @@
 splunk-otel-collector=0.34.1
 
 # Represents the current release of the OpenTelemetry Operator.
-operator=0.0.1
+operator=0.0.2
```
Comment (owais) [inline versions.txt:8]:
```suggestion
operator=0.0.1
```

File: Makefile (hunk 2 above)
Comment (owais) [inline Makefile:48]:
```suggestion
```

**Comment by owais** [comment]
> I thought that it would be better to pass as minimal information as necessary...
> Do you want me to change it?

No, that part is fine. I was more concerned about using a global var instead of a field on the struct.

**Pull request merged**